docs/source/model_design/vivarium_model_components/impairments/index.rst

```diff
@@ -33,17 +33,224 @@ Modeling Impairments
+
+In short, sometimes GBD has a lot of information about an overall condition that 
+the modelers want to utilize. However, the information typically couldn't be included in the 
+cause hierarchy since the impairment has more than one cause. Therefore, modelers 
+include it as an impairment. 
+
+An easy example to grasp is blindness. There is a lot of data available about blindness, but 
+it can be caused by a lot of different things. Therefore, it is difficult to place in the 
+cause hierarchy, but we want to include all the rich data we have. Hence, it is an 
+impairment. 
+
+Modeling impairments can be important for a simulation depending on the research questions. 
+Continuing the blindness example above, if you wanted to include an intervention that 
+targets blindness generally, you would ideally want to include blindness, but 
+wouldn't necessarily want to include all of the possible causes for blindness. 
+Hence the need to model the impairment instead of all causes of the impairment. Including 
+all of the causes would unnecessarily complicate the model and include information 
+not relevant to our research question. 
+
+On the other hand, if we were investigating an intervention that targets diabetes, 
+we might not need to include blindness as an impairment. Instead we could just track 
+the sequela "blindness due to diabetic mellitus" since we are only concerned about 
+blindness due to a single cause. 
+
+It is important to think through your research question and what is being affected 
+by your interventions to determine if including an impairment is needed. 
 
 Comparing and Contrasting Impairment and Cause Models
 -----------------------------------------------------
 
-- Most of the modeling is very similar and we can utilize the cause model page 
-- What is not applicable from that page? (e.g., certain data sources) 
+In general, modeling impairments is similar to modeling causes - they are 
+both medical conditions simulants can experience. Therefore, most of 
+the :ref:`cause modeling page <models_cause>` is still applicable and 
+should generally be used for modeling impairments. 
+
+However, there are a few important differences between modeling causes 
+and impairments that we will cover here. These differences largely 
+pertain to data sources and availability. 
+
+Disease Models
+++++++++++++++
+
+The GBD data availability differs by impairment significantly. For 
+example, heart failure has an incidence, prevalence, excess 
+mortality rate, and disability weights. Therefore, you could create a heart 
+failure model more of less exactly as you might make a cause model. 
+
+On the other hand, anemia does NOT have an incidence or any mortality 
+associated with it. It does have a prevalence and disability weight. 
+Therefore, other methods might be needed in order to have a dynamic 
+anemia model. You can see one example of this based on simulant's hemoglobin 
```

Comment:
I feel like there could be a simpler way to do this, with standard GBD assumptions: just aggregate incidence, remission, etc from all the causes, weighted by prevalence of their sequelae in the impairment. Does that sound reasonable to you? If so, should we include it as a possibility here?

Reply:
Hmm I can definitely include it. And it is an option. I think you're a bit underestimating how many causes and sequelae would be involved in this though! We use some of this data for HF and got it from the CVD team - there are a lot of causes and things involved.

Reply:
Fair enough. I didn't mean to say it would be simple! Just seems simpler than having to seek external data sources. And, most of the complexity would be on the research/data prep side so more of a 1-time cost than adding complexity to the sim itself.